config/config.exs

```diff
@@ -60,3 +60,5 @@ config :dbservice, Dbservice.Repo,
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{config_env()}.exs"
+
+config :dbservice, env: Mix.env()
```

Comment:
After pushing the code for the domain_whitelist_plug.ex file, my checks on GitHub failed with the error message Function Mix.env/0 does not exist. So here is solution to avoid this error : https://stackoverflow.com/questions/35010950/get-current-environment-name/44747870#44747870